spec/spec_helper.rb

```diff
@@ -120,7 +120,8 @@
 
   c.allow_http_connections_when_no_cassette = false
   c.default_cassette_options = {
-    :allow_unused_http_interactions => false
+    :allow_unused_http_interactions => false,
+    :decode_compressed_response     => true
```

Comment:
This concerns me as it's an across-the-board change for all providers, but I think the specs can verify if this change is acceptable or not.